app/code/Transoft/Blog/Block/Bloglist.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Why do we need this block?

Reply:
This block is used to render list of blogs on a product page.

Reply:
Please use Block or ViewModel, but not both. See app/code/Transoft/Blog/ViewModel/Bloglist.php

Reply:
fixed